osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -40,8 +41,12 @@ public class PlayerLoader : ScreenWithBeatmapBackground
 
         public override bool DisallowExternalBeatmapRulesetChanges => true;
 
+        public override bool HandlePositionalInput => true;
```

Comment:
why was this added?

Reply:
Ah this was from when I was trying to use IsHovered, as that can only be updated if I did this. Will revert.

Reply:
Actually, just realized we still need this for the PlayerLoader to know whether or not we are hovered over it to continue to the next screen. Since we were using OnHover and OnHoverLost before, this wasn't necessary. Will add a comment to explain this.